tasks/centos.yml

```diff
@@ -23,18 +23,18 @@
       set_fact:
         python3_command: python3.6
 
-    # Have to use `shell` here because `type` is a builtin.
-    # TODO: Surely there's a module for this?
     - name: Find installed executable
-      shell: "type -p {{ python3_command }}"  # noqa 305
+      find:
+        paths: /usr/bin
```

Comment:
What if python3 isn't in /usr/bin, e.g. pyenv installations?

Reply:
This is for the system install on CentOS or Ubuntu, so the installation will always be /usr/bin (right?). The current implementation has the advantage of not hard-coding a path, but it might pick up a stray python3 other than the just-installed version if this isn't a fresh system.

Reply:
True. Though Could use alternative to ensure this python3 setup is the one used.

Reply:
> What if python3 isn't in /usr/bin, e.g. pyenv installations? 

The base path could be determined within the decided installation path. So when it comes tile you have also defined the directory in which the python3 binary lives for its full path.